est/jcabi-s3.est

```diff
@@ -0,0 +1,13 @@
+date: 02-06-2015
+author: Georgy Vlasov
+method: champions.pert
+scope:
+    1: implementations of Buckets, Ockets and Regions
+    2: project site
+    3: achieve 80% test coverage
+    4: CI
+champions:
+    1:
```

Comment:
@Suseika 1 champion is not enough. We need at least 2.
Please, see the README https://github.com/teamed/est/blob/master/README.md